<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowRGBCube</class>
 <widget class="QWidget" name="ShowRGBCube">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="VSliderBrightness">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>21</width>
     <height>241</height>
    </rect>
   </property>
   <property name="maximum">
    <number>443</number>
   </property>
   <property name="pageStep">
    <number>25</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameZone">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>350</width>
     <height>350</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetColor">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>61</width>
     <height>91</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>R:128</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G:128</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B:128</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
